internal/landing/html/inspect.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title> Inspect debug data </title>
+    </head>
+    <body>
+        <form class="form-group" action="/inspect" method="POST">
+            <div class="form-group">
+                <label for="cluster-id">Enter Cluster ID :</label>
+                <input type="text" id="cluster-id" name="cluster-id" class = "form-control" placeholder="sjdhfUG@%HKA^Srasfjskfj9987sfkj"><br>    
+            </div>   
+            <button type="submit" class="btn btn-default">Submit</button>
+        </form>
+        <br><hr>
+        <h4>The affiliates associated with given cluster-id {{.ClusterId}}: </h4>
```

Comment:
```suggestion
        <h4>The affiliates associated with the given cluster ID {{.ClusterId}}: </h4>
```

Reply:
(not sure about `the`)

Reply:
should this be wrapped with `{{ if .ClusterId }}` to hide that part of the page if the form is not POSTed yet?